included the scope configuration as part of server configuration and … (#1513)
OctopusDeploy/docs: docs/security/authentication/okta-authentication.md

![Settings](okta/okta-settings.png "width=500")

:::hint

> The request to Okta from Octopus will need to include the required scopes, see [Inspect the request to Okta for scope](#Oktaauthentication-Inspecttherequesttookta) for information about how to inspect the current requests scope.

Reviewer: ```suggestion
The request to Okta from Octopus will need to include the required scopes. See the [Inspect the request to Okta for scope](#Oktaauthentication-Inspecttherequesttookta) section for information about how to inspect the scope of the current request.
```

:::

> Run the command below as Administrator to configure the scopes OpenId, Profile, Email and Groups:

Reviewer: ```suggestion
Run the command below as an Administrator to configure the scopes OpenId, Profile, Email, and Groups:
```

> The request to Okta from Octopus will need to include the required scopes, see [Inspect the request to Okta for scope](#Oktaauthentication-Inspecttherequesttookta) for information about how to inspect the current requests scope.
> :::
> 
> Run the command below as Administrator to configure the scopes OpenId, Profile, Email and Groups:
> ```

Reviewer: ```suggestion
```text
```

> ```

Reviewer: ```suggestion
```text
```
